components/user_profile/user_profile.test.tsx

```diff
@@ -4,13 +4,15 @@
 import React from 'react';
 import {shallow} from 'enzyme';
 
-import UserProfile from './user_profile';
+import {UserProfile as UserProfileType} from 'mattermost-redux/types/users';
+
+import UserProfile, {UserProfileProps} from './user_profile';
 
 describe('components/UserProfile', () => {
-    const baseProps = {
+    const baseProps: UserProfileProps = {
```

Comment:
0/5, but I don't think this should be necessary since we're specifying the type of `baseProps.user` below

Reply:
I'll double-check this. I think you are right, but I want to make sure the type checker will be happy with removing this hint.

Reply:
I can confirm this was totally unnecessary. Removing it!